### PR TITLE
Passport twitter (WIP)

### PR DESCRIPTION
**Pull request by seyiogunjuyigbe**
Twitter authentication being setup (WIP)

### CODE REVIEW

File: src/config/twitterAuth.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import SITE_URL from "./constants"
+
+export const configTwitter = {
+    'twitterAuth' : {
+        'consumerKey'       : 'gpxS6tBGRIw9biXdMT9oMVXxB',
+        'consumerSecret'    : 'RSkfOidDZUKIOJClFPBC1yY0W4kDEYQx7ASYoalYPRjAszPxKU ',
+        'callbackURL'       : `${SITE_URL}/auth/twitter/callback`
```
Comment (shonubijerry) [inline src/config/twitterAuth.js]: these should be inside `.env` and then share with the team. We cannot expose these credentials to the public.

File: src/config/passport.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+                // if the user is found then log them in
+                if (user) {
+                    return done(null, user); // user found, return that user
+                } else {
+                    // if there is no user, create them
+                    var newUser                 = new User();
+                    // set all of the user data that we need
+                    newUser.twitter.id          = profile.id;
+                    newUser.twitter.token       = token;
```
Comment (shonubijerry) [inline src/config/passport.js]: Does this field `token` exist in users table?

File: src/config/twitterAuth.js (hunk 1 above)
Comment (seyiogunjuyigbe) [reply src/config/twitterAuth.js]: Alright... done

**Comment by seyiogunjuyigbe** [comment]
Changes made.. please check

**Pull request merged**